addons/account/models/account_move_line.py

```diff
@@ -848,27 +846,16 @@ def _compute_sequence(self):
 
     @api.depends('quantity', 'discount', 'price_unit', 'tax_ids', 'currency_id')
     def _compute_totals(self):
+        AccountTax = self.env['account.tax']
         for line in self:
-            if line.display_type != 'product':
+            if line.display_type not in ('product', 'cogs'):
```

Comment:
Indeed but for some reason, price_subtotal/price_total must be computed on cogs line...

Reply:
what do you mean "must be computed on cogs line"? is there a failing test otherwise?

I suppose the price_subtotal and price_total for cogs  is always the balance of the line, no?

Reply:
:facepalm: ok, we don't have time to fix that... can you add a comment not to forget?

> #TODO check remove cogs from here and use balance instead